test/swivel/Swivel.sol

```diff
@@ -195,24 +191,26 @@ contract Swivel {
   /// @param a Array of order volume (principal) amounts relative to passed orders
   /// @param c Components of a valid ECDSA signature
   function exit(Hash.Order[] calldata o, uint256[] calldata a, Sig.Components[] calldata c) external returns (bool) {
-    for (uint256 i=0; i < o.length; i++) {
-      // Determine whether the order being filled is an exit
-      if (!o[i].exit) {
-        // Determine whether the order being filled is a vault initiate or a zcToken initiate
-          if (!o[i].vault) {
-            // If filling a zcToken initiate with an exit, one is exiting zcTokens
+    unit256 length = o.length; 
```

Comment:
its already calldata man. is this really doing anything? 

besides just being ugly

i kind of see where avoiding storage can save _some_ gas, but not avoiding _calldata_. i'd need proof here. calling bs... maybe i learn something tho

Reply:
i actually commented *exactly* this on the audit and had to prove it doing my own remix tests.

Reply:
Woops wasn't this exact change but i tested a few and all were actually minor improvements. 

(this was the one i was thinking of: https://github.com/code-423n4/2021-09-swivel-findings/issues/43)